pages/client-libraries/python.mdx

```diff
@@ -645,65 +645,128 @@ The `Session.run()` method is most commonly used for `LOAD CSV` clause to preven
+if the commit takes a second and you have 120 retries, that means that the transaction will be running for 2 minutes + waiting time.
+- `initial_wait_time` - the time that the transaction will wait before the first retry. The first time a transaction fails with conflict, it will wait for `initial_wait_time` seconds before the first retry.
+- `backoff_factor` - the factor by which the retry delay will be multiplied after each retry. This number should not be too high because it can lead to long-running transactions; if commits are small and fast, avoid exponential backoff.
+- `jitter` - the factor by which the retry delay will be randomized after each retry. If there are a lot of transactions running in parallel, it is recommended to use jitter to avoid the thundering herd problem.
 
-URI = "bolt://localhost:7687"
-AUTH = ("", "")
 
+If you use managed transactions, you can configure the retry scenario to use the `session` configuration. Here is an example: 
 
-async def main():
-    async with AsyncGraphDatabase.driver(URI, auth=AUTH) as client:
-        async with client.session(database="memgraph") as session:
-            records = await session.execute_read(get_users)
-            print(records)
+```Python
+import multiprocessing
+from neo4j import GraphDatabase
 
+HOST_PORT = "bolt://localhost:7687"
 
-async def get_users(tx):
-    result = await tx.run("MATCH (u:User) RETURN u.name AS name")
-    records = await result.values()
-    return records
+def process_chunk_managed_API(query, create_list):
+    driver = GraphDatabase.driver(HOST_PORT, auth=("", ""))
+    with driver.session(max_transaction_retry_time=180.0, initial_retry_delay=0.2, retry_delay_multiplier=1.1, retry_delay_jitter_factor=0.1) as session:
+        session.execute_write(lambda tx: tx.run(query, {"batch": create_list}))
+    driver.close()
 
+query = """
+WITH $batch AS nodes
+UNWIND nodes AS node
+MATCH (a:Node {id: node.a}), (b:Node {id: node.b}) CREATE (a)-[:RELATIONSHIP]->(b)
+"""
+
+chunks = [[{"a": i, "b": i+1} for i in range(1000)] for _ in range(10)]
+with multiprocessing.Pool(10) as pool:
+    pool.starmap(process_chunk_managed_API, [(query, chunk) for chunk in chunks])
 
-if __name__ == "__main__":
-    asyncio.run(main())
 ```
 
+In this case, the `TransientError` will be retried using the retry strategy that is configured in the `session` object. You do not need to handle the retry logic; it is handled by the driver in the background.
+
+The essential configuration arguments are the following:
+
+- `max_transaction_retry_time` is the maximum time that the transaction will be retried; after that, it will be aborted with a timeout error. 
+- `initial_retry_delay` is the time that the transaction will wait before the first retry. 
+- `retry_delay_multiplier` is the factor by which the retry delay will be multiplied after each retry.
+- `retry_delay_jitter_factor` is the factor by which the retry delay will be randomized after each retry. 
```

Comment:
```suggestion
- `max_transaction_retry_time` - the maximum time the transaction will be retried; after that, it will be aborted with a timeout error. 
- `initial_retry_delay` - the time that the transaction will wait before the first retry. 
- `retry_delay_multiplier` - the factor by which the retry delay will be multiplied after each retry.
- `retry_delay_jitter_factor` - the factor by which the retry delay will be randomized after each retry. 
```

Reply:
The same format as above